_conferences/touraine_tech_20.md

```diff
@@ -0,0 +1,12 @@
+---
+name: "Touraine Tech '20"
```

Comment:
Do we need the `'20` as part of the name?

Reply:
I removed the year as it makes no sense in title